src/main/java/net/atos/zac/aanvraag/ProductAanvraagService.java

```diff
@@ -63,9 +67,13 @@ public class ProductAanvraagService {
     @Inject
     private KVKClientService kvkClientService;
 
+    @Inject
+    private VRLClientService vrlClientService;
+
     public void verwerkProductAanvraag(final URI productAanvraagUrl) {
         final ORObject object = objectsClientService.readObject(getUUID(productAanvraagUrl));
         final ProductAanvraag productAanvraag = new ProductAanvraag(object.getRecord().getData());
+        final CommunicatieKanaal communicatieKanaal = vrlClientService.findCommunicatiekanaal("E-formulier");
```

Comment:
He, wat raar. Er zou een constante zijn op een geschiukte plaats, hadden we toch afgesproken?

Reply:
Helemaal vergeten bij het uitwerken. Opgelost in https://github.com/NL-AMS-LOCGOV/zaakafhandelcomponent/pull/1467